### ISSUE TITLE
error when editing a fitnesse page

### ISSUE THREAD

**Issue by inico** [jguglielmi/OASIS-Maven#8]
Editing any page provides an error in the page

reproduction path:
start fitnesse
open any page
click edit button
page times out and the fitnesse process contains error

 [java] WARNING: Unexpected exception [fitnesse.responders.editing.NewPageResponder.getDefaultContent(Lfitnesse/FitNesseContext;)Ljava/lang/String;]
     [java] java.lang.NoSuchMethodError: fitnesse.responders.editing.NewPageResponder.getDefaultContent(Lfitnesse/FitNesseContext;)Ljava/lang/String;
     [java] 	at org.oasis.plugin.responders.OasisEditResponder.doMakeHtml(OasisEditResponder.java:92)
     [java] 	at org.oasis.plugin.responders.OasisEditResponder.doMakeResponse(OasisEditResponder.java:65)
     [java] 	at org.oasis.plugin.responders.OasisEditResponder.makeResponse(OasisEditResponder.java:46)
     [java] 	at fitnesse.FitNesseExpediter.createGoodResponse(FitNesseExpediter.java:136)
     [java] 	at fitnesse.FitNesseExpediter.makeResponse(FitNesseExpediter.java:114)
     [java] 	at fitnesse.FitNesseExpediter.start(FitNesseExpediter.java:50)
     [java] 	at fitnesse.FitNesseServer.serve(FitNesseServer.java:28)
     [java] 	at fitnesse.FitNesseServer.serve(FitNesseServer.java:21)
     [java] 	at fitnesse.socketservice.SocketService$ServerRunner.run(SocketService.java:110)
     [java] 	at java.lang.Thread.run(Thread.java:745)

**Comment by jguglielmi** [comment]
Inico, can you please provide your pom.xml?  I'm going to troubleshoot this for you.  The way FitNesse plugins work is that there is a file called oasisplugin.  There is a side project where you can see it for yourself.  If you are using a new version of FitNesse, they may have removed some things.

**Comment by inico** [comment]
I've uploaded the pom.xml to pastebin:

http://pastebin.com/ubT85gHe

**Comment by jguglielmi** [comment]
Thanks Inico!  There were multiple issues.  The project hasn't been up-kept for a few months now, but the issues were related to the oasisplugin.jar file.  There is a project which I have just updated to handle it.  The problems were with the oasisEditPage.vm which required updating.  I have also updated the version of FitNesse and Selenium as well just to keep pace.  You will find the fitnessestandalone.jar and the individual pom files have been updated.  A lot of the source code has changed in FitNesse since 1-14-2015.  Specifically, these buggers had to be changed in the OasisPlugin project to maintain pace.  Take a fresh pull, do a Maven Update on the project, and perform the install.bat again then run it and let me know how it goes!


    util.StringUtil, use org.apache.commons.lang.StringUtils instead.
    util.ListUtility, use java.util.Arrays instead.
    fitnesse.wikitext.Utils, use fitnesse.html.HtmlUtil instead.
    fitnesse.FitFilter.


http://fitnesse.org/.FrontPage.FitNesseDevelopment.FitNesseRelease20150114
